### ISSUE TITLE
SettingsBundle requires non-existent %sylius_cache% parameter

### ISSUE THREAD

**Issue by jrobeson** [Sylius/Sylius#2684]
when trying to use SettingsBundle from master to test the SyliusResourceBundle translations decoupling PR, i ran into this:

```bash
 [Symfony\Component\DependencyInjection\Exception\ParameterNotFoundException]  
  You have requested a non-existent parameter "sylius.cache".                   
```

which occurs in the SyliusSettingsExtension class

```php
        $container->prependExtensionConfig('doctrine_cache', array(
            'providers' => array(
                'sylius_settings' => '%sylius.cache%'
            )
        ));
```

Does this simply need a hasParameter() check or will that cause a problem when using it within the full Sylius installation?

**Comment by jrobeson** [comment]
or maybe it should just default to file_system is %sylius.cache% isn't set? I guess that would be up to you folks.

**Comment by jrobeson** [comment]
@stloyd : you seem to have implemented this: https://github.com/Sylius/SyliusSettingsBundle/commit/d196af8cd44e6dd3c5ba42e82abae2b019f90512 . What do you think?

**Comment by stloyd** [comment]
It's quite strange cause settings can't work with that setting anyway cause of: https://github.com/Sylius/Sylius/commit/397c4e1f7ce94bebb6def4ebddd7604f58b3a319 but I'm ok to check in load of the settings extension to check whatever this variable exists & set if not.

**Comment by jrobeson** [comment]
@stloyd : I'm using SettingsBundle standalone, outside of the context of Sylius, so i have no %sylius.cache%

**Comment by jrobeson** [comment]
@stloyd : so are you trying to say that it should default to file_system if %sylius.cache% is not set?

**Comment by stloyd** [comment]
@jrobeson I guess that is acceptable. You can add setting to Settings extension (load section) that would be used by default like:
```php
if (!$container->hasParameter('sylius.cache')) {
    $container->setParameter('sylius.cache', 'file_system');
}

**Comment by aRn0D** [comment]
Closed by #2685

**Comment by Mrkisha** [comment]
When updated from 0.12 to 0.14 I get the same 

ParameterNotFoundException in ParameterBag.php line 106:
You have requested a non-existent parameter "sylius.cache".

Had to downgrade to 0.12.
